lib/systems/include/BaseController.h

```diff
@@ -1,16 +1,25 @@
 #ifndef BASECONTROLLER
 #define BASECONTROLLER
 #include "SharedDataTypes.h"
+
+/// @brief definition for a drivetrain command with no torque
```

Comment:
->this namespace only contains this one definition, ill create a mosre descriptie header for it tho, but this pr is only for documentation not implementation
->also its used multiple times in the code to make it more readable and explicit when its setting the drive train command to a base one

Reply:
i dont really have authority to make impleementation changes(plus it might be useless with ht09 soon?), my scope for this task is to make documentation, so i wont change any of these implementation things for the next review